landlab/components/landslides/landslide_probability.py

```diff
@@ -105,12 +145,7 @@ class LandslideProbability(Component):
 
     Create a grid on which to calculate landslide probability.
 
-    >>> grid = RasterModelGrid((5, 4), xy_spacing=(0.2, 0.2))
-
-    Check the number of core nodes.
-
-    >>> grid.number_of_core_nodes
-    6
+    >>> grid = landslide_unitgrid.build_grid_unitarea() 
```

Comment:
I'm concerned about putting all of the grid building into the landslide_unitgrid.build_grid_unitarea() function. The docstring should show potential users how to use the component, including setting up a grid the component can run on.